wlroots and seatd as subprojects causes multiple definitions error

### Expected behavior:

Everything builds without error.

### Actual behavior:

```
log.c:11: multiple definition of `NSEC_PER_SEC'; subprojects/wlroots/libwlroots.a(util_time.c.o):/home/dylan/.cache/kiss/proc/20779/build/sway-tiny/build/../subprojects/wlroots/util/time.c:7: first defined here
collect2: error: ld returned 1 exit status
```

### Workaround:

```
# Remove conflict between static libseat and wlroots.
# Workaround can also be applied to wlroots rather than seatd.
sed 's/\(const long NSEC\)/static \1/' \
    subprojects/seatd/common/log.c > _ 
mv -f _ subprojects/seatd/common/log.c
```

The conflict is between wlroots and libseat.  Was not sure where to open this issue. 
Let me know if I should open this elsewhere. This is with latest master for wlroots,
sway and seatd.

--- a/util/time.c
+++ b/util/time.c
@@ -4,7 +4,7 @@
 
 #include "util/time.h"
 
-const long NSEC_PER_SEC = 1000000000;
+static const long NSEC_PER_SEC = 1000000000;
 
 int64_t timespec_to_msec(const struct timespec *a) {
 	return (int64_t)a->tv_sec * 1000 + a->tv_nsec / 1000000;

